devlib/instrument/daq.py

```diff
@@ -55,6 +55,7 @@ def __init__(self, target, resistor_values,  # pylint: disable=R0914
                                                  resistor_values=resistor_values,
                                                  channel_map=channel_map,
                                                  labels=labels)
+        self.sample_rate_hz = sampling_rate
```

Comment:
Probably worth renaming `sampling_rate` argument to `sample_rate_hz` for consistency.